### PR TITLE
done

### PR DESCRIPTION
**Pull request by JanikOskar**
@pumanitro

### CODE REVIEW

File: Exee4.htm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <head>
+      <title>Exercise 4</title>
+    </head>
+    <body>
+      <button onclick="myFunction()">Try it</button>
+      <p id="demo"></p>
+      <script>
+        function myFunction() {
+          var n =Number(prompt("enter your number"));
+         var sum =((n * (n + 1)) / 2);
```
Comment (pumanitro) [inline Exee4.htm]: #challenge
Try to do it without math formula. But with loops.

File: Ex6.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <html>
+    <head>
+      <title>Exercise 6</title>
+    </head>
+    <body>
+      <button onclick="myFunction()">Try it</button>
+      <p id="demo"></p>
+      <script>
+        function myFunction() {
+          let l = prompt("Take s for sum or m for multi");
```
Comment (pumanitro) [inline Ex6.html]: Name variable in a such way they means something. Change l to e.g. choice.

File: index.html [hunk 3]
```diff
@@ -1,12 +0,0 @@
-<!DOCTYPE html>
```
Comment (pumanitro) [inline index.html]: You have removed first task. It wasn't described in task definition.

File: ex8.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+          let i = 3,
+            count,
+            c;
+          let n = 20;
+
+          if (n >= 1) {
+            document.write("First " + n + " prime numbers are :\n");
+          }
+          for (count = 2; count <= n; ) {
+            for (c = 2; c <= i - 1; c++) {
```
Comment (pumanitro) [inline ex8.html:21]: What this second loop do? Is it necessary?

Comment (JanikOskar) [reply ex8.html:21]: Hmm im think its necessary, without her code not working xd good english

File: ex10.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    <head>
+      <meta charset="UTF-8" ; />
+      <title>Exercise 10</title>
+    </head>
+    <body>
+      <button onclick="myFunction()">Try it</button>
+      <p id="demo"></p>
+      <script>
+        function myFunction() {
+          var year = Number(prompt("enter a year"));
```
Comment (pumanitro) [inline ex10.html:12]: Don't use var. Instead, use let or const - read why.

**Review by JanikOskar — COMMENTED** [review]
@pumanitro answer me, why?

File: ex8.html (hunk 4 above)
Comment (pumanitro) [reply ex8.html:21]: K, you are right,

**Pull request merged**